README.md

```diff
@@ -71,28 +71,25 @@ rpc:
 http:
     address: :8082
     pool:
-        num_workers: 4
-        max_jobs: 64
+        num_workers: 8
+        debug: false
```

Comment:
It doesn't make sense to write a default value

```yml
  # Workers pool settings.
  pool:
    # Debug mode for the pool. In this mode, pool will not pre-allocate the worker. Worker (only 1, num_workers ignored) will be allocated right after the request arrived.
    #
    # Default: false
    debug: false
```

Reply:
From the configure point of view, it is.
I left this option there for a better developer experience.
Additionally added a comment 👍